Formula/opencascade.rb

```diff
@@ -0,0 +1,45 @@
+class Opencascade < Formula
+  desc "3D modeling and numerical simulation software for CAD/CAM/CAE"
+  homepage "https://www.opencascade.com/content/core-technology"
+  url "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=refs/tags/V7_2_0;sf=tgz"
+  version "7.2.0"
+  sha256 "adb6fc44efebdb09e30a487fa412afae9599992dc771b0039b4ebf306891a558"
+  head "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=HEAD;sf=tgz"
+
+  depends_on "cmake" => :build
+  depends_on "freetype"
+  depends_on "freeimage"
+  depends_on "tbb"
+  depends_on "gl2ps"
+  depends_on "tcl-tk"
+  depends_on "doxygen"
```

Comment:
surely this is build time only

Reply:
Yep.  Only needed to build and install the docs.

Reply:
then `=> :build`

Reply:
done